### PR TITLE
useradd: add page

### PR DESCRIPTION
**Pull request by bigspennyj**

### CODE REVIEW

File: pages/common/useradd.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Create a user using the current default configuration:
+
+`sudo useradd {{username}}`
+
+- Create a user in the specified group with the specified home directory path:
+
+`sudo useradd -g {{group}} -d {{/path/to/home/}} {{username}}`
+
+- Create user and copy files from `/skel/dir` to user's home directory:
```
Comment (agnivade) [inline pages/common/useradd.md]: Change it to "copy files from a given dir".

File: pages/common/useradd.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`sudo useradd -k {{/skel/dir}} {{username}}`
```
Comment (agnivade) [inline pages/common/useradd.md]: {{/skel/dir}} => {{path/to/directory}}

File: pages/common/useradd.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Display the current default user configuration values:
+
+`sudo useradd -D`
+
+- Change the default home directory prefix, group, and shell for new users:
```
Comment (agnivade) [inline pages/common/useradd.md]: IMO, I don't think this option is popular enough to be included.

**Comment by agnivade** [comment]
Oops, the page for `useradd` is already there in pages/linux. I am not sure it exists for osx also. If it does, please remove the page from the linux folder and add it here. If not, you can make changes to that page if you want.

File: pages/common/useradd.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/useradd.md]: dir --> directory (or folder, if you prefer something shorter)

**Comment by bigspennyj** [comment]
Oh, not sure how I missed that!  Guess I should have done better due diligence, my bad.
I may submit a pr with changes to useradd, but the existing page looks pretty good.

**Comment by waldyrious** [comment]
Thanks for the contribution anyway, @bigspennyj :)

**Pull request closed without merging**